modules/detect_target/detect_target_factory.py

```diff
@@ -27,8 +28,25 @@ def create_detect_target(
     save_name: str,
 ) -> tuple[bool, base_detect_target.BaseDetectTarget | None]:
     """
-    Construct detect target class at runtime.
+    Factory function to create a detection target object.
+
+    Parameters:
+    detect_target_option: Enumeration value to specify the type of detection.
+    device: Target device for inference ("cpu" or CUDA device index).
+    model_path: Path to the model file.
+    override_full: Force full precision floating point calculations.
+    local_logger: Logger instance for logging events.
+    show_annotations: Whether to display annotated images.
+    save_name: Prefix for saving logs or annotated images.
+
+    Returns:
+    Tuple containing success status and the instantiated detection object (if successful).
     """
```

Comment:
cc @Xierumeng Do we want to change this docstring?

Reply:
Remove parameters, keep return text.